integration/fixtures/test-images-non-distroless.json

```diff
@@ -0,0 +1,92 @@
+        "ignoreErrors": false
+    },
+    {
+        "image": "docker.io/library/nginx",
+        "tag": "1.21.6",
+        "digest": "sha256:2bcabc23b45489fb0885d69a06ba1d648aeda973fae7bb981bafbb884165e514",
+        "distro": "Debian",
+        "description": "Valid dpkg/status, apt present",
+        "ignoreErrors": false
+    },
+    {
+        "image": "docker.io/library/nginx",
+        "tag": "1.21.6",
+        "digest": "sha256:2bcabc23b45489fb0885d69a06ba1d648aeda973fae7bb981bafbb884165e514",
+        "localName": "local/image:tag",
+        "distro": "Debian",
+        "description": "Valid dpkg/status, apt present, locally tagged with repo and image name",
+        "ignoreErrors": false
+    },
+    {
+        "image": "registry.k8s.io/kube-proxy",
+        "tag": "v1.23.4",
+        "digest": "sha256:30116c7218264d95623d3918a50da703675755cae866cd4c324586611fcd50ea",
+        "distro": "Debian",
+        "description": "Valid dpkg/status, apt present, custom network config",
+        "ignoreErrors": false
+    },
+    {
+        "image": "quay.io/calico/cni",
+        "tag": "v3.15.1",
+        "digest": "sha256:a925b445c2688fc9c149b20ea04faabd40610d3304a6efda68e5dada7a41b813",
+        "distro": "Redhat",
+        "description": "Valid rpm DB, microdnf & rpm present",
+        "ignoreErrors": false
+    },
+    {
+        "image": "mcr.microsoft.com/cbl-mariner/base/core",
+        "tag": "2.0.20240112",
+        "digest": "sha256:60323975ec3aabe1840920a65237950a54c5fef6ffc811a5d26bb6bd130f1cc3",
+        "distro": "Mariner",
+        "description": "Valid rpm DB, no dnf, yum & rpm present",
+        "ignoreErrors": false
+    },
+    {
+        "image": "mcr.microsoft.com/cbl-mariner/base/core",
+        "tag": "2.0.20240112-arm64",
+        "digest": "sha256:c85680df0ddccfd5bf0cd60ff7d0c07b0ea783bcee9ce5dc748b68c0d36e280a",
+        "distro": "Mariner",
+        "description": "Valid rpm DB, no dnf, yum & rpm present, arm64 cross-arch",
+        "ignoreErrors": false
+    },
+    {
+        "image": "docker.io/library/centos",
+        "tag": "7.6.1810",
+        "digest": "sha256:62d9e1c2daa91166139b51577fe4f4f6b4cc41a3a2c7fc36bd895e2a17a3e4e6",
+        "distro": "CentOS",
+        "description": "Valid rpm DB, yum present",
+        "ignoreErrors": false
+    },
+    {
+        "image": "docker.io/library/amazonlinux",
+        "tag": "2.0.20210326.0",
+        "digest": "sha256:06380711d6a8ac0b6989f7e2a4419e560796791d9c7c843753a719c73552dc30",
+        "distro": "Amazon Linux",
+        "description": "Valid rpm DB, yum present",
+        "ignoreErrors": false
+    },
+    {
+        "image": "docker.io/grafana/grafana-image-renderer",
+        "tag" : "3.4.0",
+        "digest": "sha256:205a39f5b58f96b9ff81a0b523a60c26c86e88e76575696fcd6debde9de02197",
+        "distro": "Alpine",
+        "description": "Valid apk/db, apk present, fail to patch libssl/libcryto",
+        "ignoreErrors": true
```

Comment:
do we need ignore errors still

Reply:
we don't need this anymore, changed to false!

Reply:
@ashnamehrotra is this shared between update all and scanner input? if so, i would expect this to still give an error if scanner input is provided

Reply:
looks like it's not shared